site/_docs_1_0/ingressroute.md

```diff
@@ -130,7 +130,7 @@ ingressroute "basic" deleted
 
 ## IngressRoute API Specification
 
-There are a number of [working examples](https://github.com/projectcontour/contour/tree/master/examples/example-workload/ingressroute) of IngressRoute objects in the `examples/example-workload` directory.
+There are a number of [working examples](https://github.com/projectcontour/contour/blog/{{ site.github.latest_release.tag_name }}/examples/example-workload/ingressroute) of IngressRoute objects in the `examples/example-workload` directory.
```

Comment:
This resolves to `https://github.com/projectcontour/contour/blog/v0.15.3/examples/example-workload/ingressroute` which is a 404.